packages/@lwc/engine/src/framework/decorators/api.ts

```diff
@@ -6,12 +6,14 @@
  */
 import assert from '../../shared/assert';
 import { isRendering, vmBeingRendered, isBeingConstructed } from '../invoker';
-import { isObject, toString, isFalse } from '../../shared/language';
-import { valueObserved, valueMutated } from '../../libs/mutation-tracker';
+import { isObject, isTrue, toString, isFalse } from '../../shared/language';
+import { valueObserved, valueMutated, ReactiveObserver } from '../../libs/mutation-tracker';
 import { ComponentInterface, ComponentConstructor } from '../component';
-import { getComponentVM } from '../vm';
+import { getComponentVM, rerenderVM } from '../vm';
 import { isUndefined, isFunction } from '../../shared/language';
 import { getDecoratorsRegisteredMeta } from './register';
+import { addCallbackToNextTick } from '../utils';
+import { ENABLE_REACTIVE_SETTER } from '@lwc/features';
```

Comment:
@ekashida in vscode, I see this import marked as invalid export for some reason.